### PR TITLE
Translate contribute/style/_index.md in Korean

### LINKED ISSUE #20864: Translate contribute/style/_index.md in Korean
**This is a Feature Request**

**What would you like to be added**

Translate contribute/style/_index.md in Korean.

**Why is this needed**

There is no document of contribute/style/_index.md which translated into Korean.
Additionally, translated contribute/style/write-new-topic.md file is displayed on the main menu instead of under style menu due to absense of style/_index.md file. 

**Comments**

Page to translate: https://kubernetes.io/docs/contribute/style/

### PR DESCRIPTION
**Pull request by pjhwa**
This fixes #20864 .

This PR contains the following file which translated into Korean.
- content/ko/docs/contribute/style/_index.md

This PR also fixes the following issue.
- Additionally, translated contribute/style/write-new-topic.md file is displayed on the main menu instead of under style menu due to absense of style/_index.md file.

### CODE REVIEW

**Comment by pjhwa** [comment]
Deploy preview: https://deploy-preview-20865--k8s-dev-ko.netlify.app/ko/docs/contribute/style/

File: content/ko/docs/contribute/style/_index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: 문서 스타일 개요
+main_menu: true
+weight: 80
+---
+
+이 섹션의 주제는 문서 작성 스타일, 컨텐츠 형식과
+구성, 쿠버네티스 문서에 맞게 사용자 정의한 Hugo의 사용에 대한
+가이드를 제공한다.
```
Comment (seokho-son) [inline content/ko/docs/contribute/style/_index.md:9]: How about this ?
```suggestion
이 섹션의 주제는 문서 작성 스타일, 컨텐츠 형식과
구성, 쿠버네티스 문서화에 적합하게 사용자 정의된 Hugo 사용 방법에 대한
가이드를 제공한다.
```

Comment (pjhwa) [reply content/ko/docs/contribute/style/_index.md:9]: Yes, much better. Thank you!

**Comment by pjhwa** [comment]
@seokho-son 리뷰 감사합니다!
의견주신대로 반영하여 업데이트 하였습니다.

**Pull request merged**